src/components/InputSearch/IInputProps.types.ts

```diff
@@ -0,0 +1,7 @@
+import { HTMLProps } from 'react';
+
+export interface IInputProps extends HTMLProps<HTMLInputElement> {
+  errors?: boolean;
```

Comment:
é um boolean, true or false. Não tem plural aqui.
```suggestion
  error?: boolean;
```

Reply:
Se voce responder na aba de files, vai ficar assim

Reply:
Feito!